ship/helper_test.go

```diff
@@ -19,6 +19,21 @@ func TestJsonFromEEBUSJson(t *testing.T) {
 	}
 }
 
+func TestJsonFromEEBUSJsonTrailingZeros(t *testing.T) {
```

Comment:
I'd love to have a comment above the test function on what this test does. E.g. something like 

```go
// The PMCP device mistakenly adds an `0x00` byte at the end of many messages. Test if this is handled correctly
```